gio/Cargo.toml

```diff
@@ -41,6 +41,7 @@ once_cell = "1.0"
 futures-core = { version = "0.3", default-features = false }
 futures-channel = "0.3"
 futures-io = "0.3"
+futures-util = "0.3"
```

Comment:
You could go with `default-features = false` here. Only the `std` feature should be needed, if at all.